library/r-00001-the-interface-will-support-gui-web-access-to-end-users.md

```diff
@@ -1,3 +1,23 @@
 ---
-title: The interface will support GUI web access to end users
+layout: default
+title: "R00001 The interface will support GUI web access to end users &#x2705; "
```

Comment:
Should the title include the number R00001? usualy not I think.

Reply:
it should not

Reply:
titles also do not need quotes ""